src/routes/blog/post/How-to-put-privacy-first/+page.markdoc

```diff
@@ -0,0 +1,86 @@
+
+## But first, what are pixels, tags, and cookies?
+
+**Pixels**, **tags**, and **cookies** are all pieces of technology used to [track user activity online.](https://www.osano.com/articles/privacy-cookies-pixels-and-tags) They are often used for marketing and advertising purposes but can also be used to improve website functionality and user experience.
+
+- **Tags** are snippets of code added to a website to track your activity and collect data. For example, a tag can be used to track how many people visit a particular page on a website or to collect information about what products you are viewing. The most well-known tool for this is [Google Tag Manager](https://www.semrush.com/blog/beginners-guide-to-google-tag-manager/). It allows you to manage all of your website tags in one place.
+- **Pixels** are small, transparent images that are embedded in a website. When you visit a website with a pixel, your browser sends information about your visit to the company that owns the pixel. This information can include the user's IP address, browser type, and the URL of the page you’re visiting. It is a beneficial tool to retarget website visitors with banner or social ads.
+- **Cookies,** first-party are set by the website you’ve visited mainly to improve your experience on that website, whereas different websites, such as Google Analytics set third-party cookies. Third-party cookies are the ones that are often used to track you across platforms, mostly meant for advertising purposes.
+
+![Appwrite is GDPR compliant](/images/blog/gdpr.png)
+
+## So, what did we remove?
+
+If you visit the Appwrite Console or website, you will see a cookie consent banner. However, you will see only two options for you to pick from, `Strictly necessary cookies` and `Product analytics.`We have removed all of our `Marketing` classified pixels, tags, and cookies such as:
+
+- HubSpot tag
+- Google Tag Manager
+- Twitter Pixel
+- Facebook Pixel
+- Google Ads Pixel
+- Google Analytics Pixel
+
+Most were used to get website analytics, analyze website clicks, measure ad performance, and collect form submissions.
+
+## The consequences of removing marketing pixels, tags, and cookies
+
+The above mentioned allowed us to analyze user behavior, measure the effectiveness of ad campaigns, retarget ads, and, to some extent, personalize the user experience. There are hardly any growth or marketing teams that don’t rely on them, and as stated in [this article from Osano](https://www.osano.com/articles/marketer-friendly-privacy-software), privacy regulations make it a lot harder for growth and marketing teams to do their work. Go to any privacy footer on any website, and you will read an essay on all the data that is being tracked. But this is precisely what we are challenging: does removing pixels, cookies, and tags make growth’s life harder? Let’s take a look.
```

Comment:
`this article from Osano` to just `article from Osano`.

Clickable texts should make sense out of context as well. `this` is giving it context.

If you really want to keep `this`, we can do it before link. But I don't think its nessessary.